modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,26 +179,34 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
 ----
++
+where:
```

Comment:
up to you, but for a "where" that is a single item you can do it all in one line like:

```suggestion
where `<num>` specifies the worker node number.
```

Reply:
Updated as suggested.